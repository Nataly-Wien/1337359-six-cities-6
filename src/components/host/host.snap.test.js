import React from 'react';
import {render} from '@testing-library/react';
import Host from './host';

it(`Should hotel host render correctly`, () => {
  const description = `In the center of Paris, is a 10-minute walk from Notre-Dame Cathedral and 293 m from Saint-Michel Metro Station. It features a fitness room and free Wi-Fi access.`;
  const host = {
    id: 1,
    name: `Mick`,
    isPro: true,
    avatarUrl: `img/avatar.jpg`
  };

  const {container} = render(<Host description={description} host={host} />);

  expect(container).toMatchSnapshot();
});
