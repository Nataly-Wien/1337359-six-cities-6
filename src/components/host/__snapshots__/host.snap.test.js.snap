// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should hotel host render correctly 1`] = `
<div>
  <div
    class="property__host"
  >
    <h2
      class="property__host-title"
    >
      Meet the host
    </h2>
    <div
      class="property__host-user user"
    >
      <div
        class="property__avatar-wrapper property__avatar-wrapper--pro user__avatar-wrapper"
      >
        <img
          alt="Host avatar"
          class="property__avatar user__avatar"
          height="74"
          src="img/avatar.jpg"
          width="74"
        />
      </div>
      <span
        class="property__user-name"
      >
        Mick
      </span>
    </div>
    <div
      class="property__description"
    >
      <p
        class="property__text"
      >
        In the center of Paris, is a 10-minute walk from Notre-Dame Cathedral and 293 m from Saint-Michel Metro Station. It features a fitness room and free Wi-Fi access.
      </p>
    </div>
  </div>
</div>
`;
